scripts/processors/cams_processor.py

```
@@ -0,0 +1,343 @@
+"""CAMS data processor - converts NetCDF files to CSV with H3 aggregation."""
+
+import logging
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)